cypress/support/generate.js

```diff
@@ -0,0 +1,15 @@
+import { faker } from '@faker-js/faker';
+
+function generateWorker() {
+  const username = faker.internet.userName();
+  const firstName = faker.name.firstName();
+  const lastName = faker.name.lastName();
+  const email = `${username}@mail.com`;
```

Comment:
The way you're generating the email address might lead to invalid emails. The `faker` library has a method `faker.internet.email()` that generates a random email address. It might be a better choice to use this method instead of concatenating the username with a static string. This would ensure that the email addresses generated are always valid and follows the proper email format.